### DISCUSSION TITLE
Add s2 support for parquet-go

### DISCUSSION THREAD

**Discussion by tchaton** [klauspost/compress#848] [Q&A]
Hey there,

I wondered how hard would it be to add s2 support for parquet-go: [parquet-go](https://github.com/xitongsys/parquet-go)

Best,
T.C

**Comment by klauspost** [comment]
So we cannot add it to the format itself.

But using the Snappy compatible compressors for `CompressionCodec_SNAPPY` should be quite easy. It would also eliminate the need for `github.com/golang/snappy v0.0.4` as a dependency.

**Reply by klauspost** [reply]
Sent in this: https://github.com/xitongsys/parquet-go/pull/553
